Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -415,7 +415,7 @@ void CGameListCtrl::InsertItemInReportView(long _Index)
 	if (gameini.GetIfExists("EmuState", "Title", &title))
 		name = title;
 
-	if (rISOFile.IsDiscTwo() && name.find("Disc 2") == std::string::npos)
+	if (rISOFile.IsDiscTwo() && (name.Lower().find("disc 2") == std::string::npos && name.Lower().find("disc2") == std::string::npos))
```

Comment:
Why are there parentheses around the disc 2 and disc2 checks combined? Having one set of parentheses around each check would make more sense, but having parentheses might not be necessary at all.